ui/pages/swaps/prepare-swap-page/review-quote.js

```diff
@@ -280,7 +281,8 @@ export default function ReviewQuote({ setReceiveToAmount }) {
   const unsignedTransaction = usedQuote.trade;
   const { isGasIncludedTrade } = usedQuote;
   const isSmartTransaction =
-    currentSmartTransactionsEnabled && smartTransactionsOptInStatus;
+    useSelector(getSmartTransactionsPreferenceEnabled) &&
+    currentSmartTransactionsEnabled;
```

Comment:
What's the reason for not using `getIsSmartTransaction` from the selectors here?

Reply:
Even though the variable is called `isSmartTransaction`, the logic behind it `(useSelector(getSmartTransactionsPreferenceEnabled) && useSelector(getCurrentSmartTransactionsEnabled))` isn’t obviously equivalent to `useSelector(getIsSmartTransaction)` at first glance. 

This PR was mainly intended as a simple rename, but if you have any suggestions, I’d be happy to include them!